Grasp:
  head_look_around:
  
    joint_names: [head_1_joint, head_2_joint]
    points:
    - positions: [0, 0]
      time_from_start: 0.1
    - positions: [0, -0.7]
      time_from_start: 3.0
    - positions: [-0.7, -0.7]
      time_from_start: 6.0
    - positions: [-0.7, -0.5]
      time_from_start: 9.0
    - positions: [0, -0.5]
      time_from_start: 12.0
    - positions: [0.7, -0.5]
      time_from_start: 15.0
    - positions: [0.7, -0.7]
      time_from_start: 18.0
    - positions: [0, -0.7]
      time_from_start: 21.0
    - positions: [0, -0.5]
      time_from_start: 24.0
